app/controllers/paypal_accounts_controller.rb

```diff
@@ -101,7 +101,8 @@ def ask_order_permission
       flash[:error] = t("paypal_accounts.new.could_not_fetch_redirect_url")
       return redirect_to action: :new
     else
-      return redirect_to permissions_url
+      render json: {redirect_url: permissions_url,
+                    redirect_message: "#{t("paypal_accounts.redirect_message")} <a href=#{permissions_url}>#{t("paypal_accounts.redirect_here")}</a>"}
```

Comment:
I feel that we shouldn't send HTML in the JSON content. Could we send the redirect_url (as we are actually already sending it) and the translation separately, and then construct the HTML in the view (JS) layer?